gluten-ut/spark32/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -184,6 +184,8 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude("Shuffle")
     // TODO: ArrayDistinct should handle duplicated Double.NaN
     .excludeByPrefix("SPARK-36741")
+    // TODO: ArrayIntersect should handle duplicated Double.NaN
+    .excludeByPrefix("SPARK-36754")
```

Comment:
@rui-mo Could you help verify if the handling of NaN in array-related functions in Velox is different from Spark?

Reply:
@Ma-Jian1 Could you help check? I remember you have some findings here.

Reply:
Can this pr be merged?

Reply:
Yes, let's merge this PR first.